cudnnModule.hpp:70:73: error: converting to ‘std::tuple<int, int, int>’ from initializer list...

I get this error when trying to build ArrayFire 3.7

```
In file included from /.../arrayfire_3.7/src/backend/cuda/cudnn.hpp:13:0,
                 from /.../arrayfire_3.7/src/backend/cuda/convolve.cpp:15:
/.../arrayfire_3.7/src/backend/cuda/cudnnModule.hpp: In member function ‘std::tuple<int, int, int> cuda::cudnnModule::getVersion()’:
/.../arrayfire_3.7/src/.../cudnnModule.hpp:70:73: error: converting to ‘std::tuple<int, int, int>’ from initializer list would use explicit constructor ‘constexpr std::tuple< <template-parameter-1-1> >::tuple(_UElements&& ...) [with _UElements = {int&, int&, int&}; <template-parameter-2-2> = void; _Elements = {int, int, int}]’
     std::tuple<int, int, int> getVersion() { return {major, minor, patch}; }
```

Environment: 
- Arrayfire 3.7
- CUDA 9.2
- cuDNN 7.6.5
- boost 1.72
- gcc 5.4.0

--- a/src/backend/cuda/cudnnModule.hpp
+++ b/src/backend/cuda/cudnnModule.hpp
@@ -67,7 +67,9 @@ class cudnnModule {
     spdlog::logger* getLogger();
 
     /// Returns the version of the cuDNN loaded at runtime
-    std::tuple<int, int, int> getVersion() { return {major, minor, patch}; }
+    std::tuple<int, int, int> getVersion() {
+        return std::make_tuple(major, minor, patch);
+    }
 };
 
 cudnnModule& getCudnnPlugin();

